packages/core/src/App/Containers/RealAccountSignup/currency-selector.jsx

```diff
@@ -152,27 +152,27 @@ class CurrencySelector extends React.Component {
     };
 
     render() {
-        const { has_currency, has_real_account } = this.props;
+        const { has_currency, has_real_account, bypass_to_personal } = this.props;
         const { is_deriv_crypto } = this.context;
 
+        // In case of form error bypass to update personal data
+        if (bypass_to_personal) this.props.goToNextStep();
```

Comment:
When `bypass_to_personal` is going to be `true`?

Reply:
It sets to `true`, once user has previous data which happens whenever user faces an error such as invalid phone number. So it will redirect the user to next step which is Personal Details.